tests/utils/verification/volumeproperties.go

```diff
@@ -109,17 +109,27 @@ func GetFullVolumeName(hostName string, volumeName string) string {
 // shorter name without @datastore suffix.
 func VerifyAttachedStatus(name, hostName, esxName string) bool {
 	log.Printf("Confirming attached status for volume [%s]\n", name)
+	return testAttachedStatus(name, hostName, esxName, esx.RetrieveVMNameFromIP(hostName))
+}
+
+// VerifyAttachedStatus - verify volume is attached and name of the VM attached
```

Comment:
Done. For now I'm not changing all the call sites, that is a separate issue to review all the test code and replace as needed.